package.json

```
@@ -30,7 +30,8 @@
     "update:emulator": "flow project deploy --update",
     "update:testnet": "npx dotenv-cli -e .env.testnet -- flow project deploy --network=testnet -f flow.json -f flow.testnet.json --update",
     "dev:reset": "cross-env node .ki-scripts/reset.js",
-    "postinstall": "npx lerna exec npm install"
+    "postinstall": "npx lerna exec npm install",
+    "stop": "npx pm2 stop dev-wallet emulator api web"
```

```suggestion
    "stop": "npx pm2 stop all"
```

Might be better to stop all just to be safe?